### PR TITLE
 Reset cluster expiration email flag after cluster lifetime extension

### LINKED ISSUE #1101: Email server shutdown warning again after extending cluster
Hey, would it be possible to send another hour warning email if a user extends the cluster life?

### PR DESCRIPTION
**Pull request by sunahsuh**
Fixes #1101 -- sets the cluster expiration email flag to false when the lifetime is extended

Also includes a revert of #1476, which it looks like *should* have been in aa4fe07 according to the extended commit message but the actual diff doesn't seem to include it. That change was necessary to get the tests to run.

### CODE REVIEW

File: requirements/tests.txt [hunk 1]
```diff
@@ -61,10 +61,9 @@ pytest-django==3.4.4 \
 pytest-mock==1.10.0 \
     --hash=sha256:53801e621223d34724926a5c98bd90e8e417ce35264365d39d6c896388dcc928 \
     --hash=sha256:d89a8209d722b8307b5e351496830d5cc5e192336003a485443ae9adeb7dd4c0
-pycodestyle==2.4.0 \
-    --hash=sha256:74abc4e221d393ea5ce1f129ea6903209940c1ecd29e002e8c6933c2b21026e0 \
-    --hash=sha256:cbc619d09254895b0d12c2c691e237b2e91e9b2ecf5e84c26b35400f93dcfb83 \
-    --hash=sha256:cbfca99bd594a10f674d0cd97a3d802a1fdef635d4361e1a2658de47ed261e3a # pyup: <2.4.0,>=2.0.0
+pycodestyle==2.3.1 \
+    --hash=sha256:682256a5b318149ca0d2a9185d365d8864a768a28db66a84a2ea946bcc426766 \
+    --hash=sha256:6c4245ade1edfad79c3446fadfc96b0de2759662dc29d07d80a6f27ad1ca6ba9 # pyup: <2.4.0,>=2.0.0
```
Comment (robhudson) [inline requirements/tests.txt]: I'm curious why this is here? The old comment was wrong also since it was `<2.4.0`

Comment (sunahsuh) [reply requirements/tests.txt]: the comment? not sure -- I just did a `git revert` on the commit that bumped the version so this is what was there before

Comment (robhudson) [reply requirements/tests.txt]: Ah, just noticed your comment in the PR. Nevermind. :)

**Comment by sunahsuh** [comment]
@jezdez @robhudson Re-upping this: is there anyone that I should notify before merging this and cutting a new release?

**Comment by robhudson** [comment]
I forget how deploys happen, so I would look towards @jezdez for that.

**Comment by jezdez** [comment]
@sunahsuh Sorry for the delay, I'll try to get this deployed this week.

**Pull request merged**